source/blender/gpu/GPU_material.h

```diff
@@ -396,6 +396,12 @@ void GPU_ambient_update_color(const float color[3]);
 void GPU_zenith_update_color(const float color[3]);
 void GPU_update_exposure_range(float exp, float range);
 void GPU_update_envlight_energy(float energy);
+float *GPU_horizon_color();
```

Comment:
I prefer using `const float` instead of `float` to avoid anyone writing on the array via its pointer.